src/gateways/PrismaGouvernanceLoader.ts

```diff
@@ -31,24 +31,38 @@ function transform(gouvernanceRecord: DepartementRecord): UneGouvernanceReadMode
         dateProchainComite: new Date('2024-09-06'),
         nom: 'Comité stratégique 2',
         periodicite: 'Semestriel',
+        type: 'stratégique',
       },
       {
         dateProchainComite: new Date('2024-08-01'),
         nom: 'Comité stratégique 1',
         periodicite: 'Trimestriel',
+        type: 'technique',
       },
     ],
     departement: gouvernanceRecord.nom,
     feuillesDeRoute: [
       {
-        budgetGlobal: 145000,
+        beneficiairesSubvention: [{ nom: 'Préfecture du Rhône', roles: ['Porteur'], type: 'Structure' }, { nom: 'CC des Monts du Lyonnais', roles: ['Porteur'], type: 'Structure' }],
+        beneficiairesSubventionFormation: [{ nom: 'Préfecture du Rhône', roles: ['Porteur'], type: 'Structure' }, { nom: 'CC des Monts du Lyonnais', roles: ['Porteur'], type: 'Structure' }],
+        budgetGlobal: 145_000,
+        montantSubventionAccorde: 5_000,
+        montantSubventionDemande: 40_000,
+        montantSubventionFormationAccorde: 5_000,
         nom: 'Feuille de route inclusion 1',
+        porteur: { nom: 'Préfecture du Rhône', roles: ['Co-orteur'], type: 'Administration' },
         totalActions: 3,
       },
       {
-        budgetGlobal: 88030,
+        beneficiairesSubvention: [{ nom: 'Préfecture du Rhône', roles: ['Porteur'], type: 'Structure' }, { nom: 'CC des Monts du Lyonnais', roles: ['Porteur'], type: 'Structure' }],
```

Comment:
```suggestion
        beneficiairesSubvention: [],
```
Histoire d'afficher le tiret et le singulier